lib/modules/summarization/models/open_ai.rb

```diff
@@ -47,13 +47,24 @@ def summarize_with_truncation(contents, opts)
           completion(messages)
         end
 
+        def summarize_single(chunk_text, opts)
+          summarize_chunk(chunk_text, opts.merge(single_chunk: true))
+        end
+
         private
 
         def summarize_chunk(chunk_text, opts)
+          summary_instruction =
+            if opts[:single_chunk]
+              "Summarize the following forum discussion, creating a cohesive narrative:"
+            else
+              "Summarize the following in 400 words:"
```

Comment:
Do we need an "up to" or this only happening when there are >1 chunks means 400 is always reasonable?

Reply:
I don't think we need it because we re-summarize the 400-word summaries.